src/hypofuzz/hypofuzz.py

```diff
@@ -465,8 +472,20 @@ def fuzz_several(*targets_: FuzzProcess, random_seed: Optional[int] = None) -> N
 
 
 @lru_cache
-def where_am_i() -> WorkerT:
-    """Return a json blob identifying the machine running this code.
+def _git_head() -> Optional[str]:
+    try:
+        return subprocess.check_output(
+            ["git", "rev-parse", "HEAD"],
+            timeout=10,
+            text=True,
+        ).strip()
+    except Exception:
+        return None
```

Comment:
I'd like to handle the case where you run `hypothesis fuzz -- path/to/repo/tests/`.  Probably the way to do that is to make this a function of `Path -> str | None`, and have each worker run it for the directory containing the test file?  This also complicates `WorkerIdentity` a bit, since a given process could be fuzzing tests from multiple git repos.

Happy to defer that to a follow-up PR though.

Reply:
I'm not too worried about the multiple-repos case; worst case we include the repository name or path and ignore our git heuristics if not in the same repo.